slave.mk

```diff
@@ -27,12 +27,12 @@ SRC_PATH = src
 RULES_PATH = rules
 TARGET_PATH = target
 DOCKERS_PATH = dockers
-BLDENV = $(shell lsb_release -cs)
+BLDENV := $(shell lsb_release -cs)
```

Comment:
>:= [](http://example.com/codeflow?start=7&length=2)

Is it fixing a bug? Is it really needed?

Reply:
This is just an optimization, so that `lsb_release -cs` is evaluated only once instead of every single time that `BLDENV` is referenced. For native arch builds, the change isn't really noticeable (maybe a second or two), but for arm64 builds on amd64, it appears to reduce the time it takes to run `make configure` by 2 minutes for each distro.